src/styles/utils/variables.scss

```
@@ -0,0 +1,9 @@
+$cards-column-count: 1;
+$color-white: #fff;
+$color-black: #000;
+$color-card-title-hover: #34568b;
+$color-card-border: #F3F3F3;
+$color-blue-accent: #00ACDC;
+$color-main-accent: #060B35;
+$color-secondary: #616070;
+$linkcolor: #00acdc;
```

Do you need two colors with the same values?
```suggestion
$color-blue-accent: #00ACDC;  // if you have here this color
$color-main-accent: #060B35;
$color-secondary: #616070;
```